fixes #1235

spuppala-rythmos opened the pull request:
Added single line namespace support

we have introduced the option nl_namespace_two_to_one_liner which turns two liner namespace to one liner.

spuppalaa-rythmos committed:
fixes #1235

Added single line namespace support

we have introduced the option nl_namespace_two_to_one_liner which turns two liner namespace to one liner.

hschlichter reviewed (changes requested):
The diff per-say is fine. But I think the option should be disabled by default. Thereby not yielding a diff by the unity-diff-checker.

spuppalaa-rythmos committed:
As suggested by Henrik disabled the option and default behaviour

option  :
nl_namespace_two_to_one_liner=false
Default behaviour :
converting 2 liners to four lines

spuppalaa-rythmos committed:
Merge branch 'master' into fixes/add-single-line-namespace-support

spuppalaa-rythmos committed:
option num incrimented
